app-office/libreoffice/libreoffice-9999.ebuild

```diff
@@ -25,7 +27,10 @@ inherit autotools bash-completion-r1 check-reqs flag-o-matic java-pkg-opt-2 mult
 
 DESCRIPTION="A full office productivity suite"
 HOMEPAGE="https://www.libreoffice.org"
-SRC_URI="branding? ( https://dev.gentoo.org/~dilfridge/distfiles/${BRANDING} )"
+SRC_URI="
+	branding? ( https://dev.gentoo.org/~dilfridge/distfiles/${BRANDING} )
+	qrcode? ( https://dev-www.libreoffice.org/src/${MY_BUNDLED_QR}.tar.gz )
```

Comment:
Please check `download.lst` for QRCODEGEN_TARBALL, add it to ADDONS_SRC instead.

Reply:
Fixed locally.